docs/GettingStartedDocs/Windows_vscode.md

```diff
@@ -0,0 +1,117 @@
+# Building And Debugging Using Visual Studio Code
+
+This document provides a brief overview of how to build and debug Open Enclave applications using VS Code.
+
+## Install VS Code
+
+The latest version of Visual Studio Code can be installed from [https://code.visualstudio.com/](https://code.visualstudio.com/)
+
+
+## Install VS Code Extensions
+
+Install the following VS Code extensions. Click on an image to navigate to the Visual Studio Code Marketplace page for the extension.
+
+[![C/C++ Extension](images/VSCodeCppExtension.png)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
+
+[![CMake Tools Extension](images/VSCodeCMakeToolsExtension.png)](https://marketplace.visualstudio.com/items?itemName=vector-of-bool.cmake-tools)
+
+[![CDB Extension](images/VSCodeCDBExtension.png)](https://marketplace.visualstudio.com/items?itemName=MicrosoftDebuggingPlatform.vscode-cdb)
+
+## Launch Visual Studio Code
+Open an instance of x64 Native Tools Command Prompt
+
+![x64 Native Tools Command Prompt](images/VSCodeNativeToolsPrompt.png)
```

Comment:
Hi Anand, you can omit setting OpenEnclave_DIR and set it in cmake.configureArgs with -DOpenEnclave_DIR.

Reply:
Great suggestion! It works. Will update the docs.

Reply:
Fixed.